src/npctalk.cpp

```diff
@@ -338,6 +365,17 @@ void game::chat()
         g->u.add_msg_if_player( m_warning, _( "You can't speak without your face!" ) );
         return;
     }
+    std::vector<vehicle *> animal_vehicles;
+    std::vector<vehicle *> following_vehicles;
+    for( auto &veh : g->m.get_vehicles() ) {
+        auto &v = veh.v;
+        if( v->has_engine_type( fuel_type_animal, false ) ) {
```

Comment:
There is no owner check here like in `assign_veh_to_follow`, is that intended? (The player can issue the command even if there is no suitable vehicle around, only non-owned vehicles with animals may be around.)